crates/rattler/src/install/python.rs

```diff
@@ -7,7 +7,7 @@ use std::path::{Path, PathBuf};
 #[derive(Debug, Clone)]
 pub struct PythonInfo {
     /// The major and minor version
-    pub short_version: (usize, usize),
+    pub short_version: (u64, u64),
```

Comment:
I want to live to see Python 18446744073709551615 :D

Reply:
Haha yeah, it's what's stored in a Version because you can have: 202306221907 (year month day hour minute) which is bigger than the largest u32.